Contact/Controller/Index/Post.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Gorilla\Contact\Controller\Index;
+
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\Action\Context;
+use Gorilla\Contact\Model\ContactFactory;
+
+class Post extends Action
+{
+
+    protected $_contactFactory;
+
+    /**
+     * Result constructor.
+     * @param Context $context
+     * @param ContactFactory $contactFactory
+     */
+    public function __construct(
+        Context $context,
+        ContactFactory $contactFactory
+    )
+    {
+        parent::__construct($context);
+        $this->_contactFactory = $contactFactory;
+    }
+
+    /**
+     * The controller action
+     *
+     * @return Redirect
+     */
+    public function execute()
+    {
+        $post = $this->getRequest()->getPost();
+        $model = $this->_contactFactory->create();
+        if (!empty($post)) {
+            try{
+                $model->setFirstName($post['first_name']);
+                $model->setLastName($post['last_name']);
+                $model->setEmail($post['email']);
+                $model->setTime(date("Y-m-d H:i:s"));
+                $model->save();
```

Comment:
Using ->save() in the model is not best practice. A repository should be defined instead.